### PR TITLE
Child health update

### PR DESCRIPTION
**Pull request by lucindalawrie**
Hi both,

I've just updated the child health data so that I can preview the s-curves to Susanne.  I wanted to get a second pair of eyes to make sure I haven't missed anything as it feels like forever since I last did anything to the dashboard!

Something I do need a little help with is adding a couple new colours to the child health palette. I don't know how to generate/pick new hex colour codes...

I'm not expecting this to be merged with the master as I have changed the ui script so I can deploy a preview with only the child health data.

Thanks,
Lucinda

### CODE REVIEW

**Comment by vaelliott** [comment]
hi lucinda, i can take a look at this

File: app_data_preparation.R [hunk 1]
```diff
@@ -767,16 +765,12 @@ sixtoeight %<>% left_join(hb_lookup, by = c("geography" = "hb_cypher")) %>%
          cohort=factor(cohort,levels=c("weekly","monthly","yearly"))) %>%
   arrange(cohort) %>%
   select (extract_date, review, week_6_start, time_period_eligible, tabno, surv, interv, cohort, area_name, area_type, week_no) %>% 
-  filter(interv<168) %>%
-  filter(substr(time_period_eligible,5,10) != "02-MAR") %>% 
-  filter(substr(time_period_eligible,5,10) != "09-MAR") %>% 
-  filter(substr(time_period_eligible,5,10) != "16-MAR") %>% 
-  filter(substr(time_period_eligible,5,10) != "23-MAR")
+  filter(interv<168)
```
Comment (vaelliott) [inline app_data_preparation.R:768]: maybe add a comment to this filter just so its explicit what this filter is about? i'm guessing we are selecting any records where the number of days is <168 (or 24 weeks?)

File: app_data_preparation.R [hunk 2]
```diff
@@ -809,16 +803,12 @@ thirteen %<>% left_join(hb_lookup, by = c("geography" = "hb_cypher")) %>%
          cohort=factor(cohort,levels=c("weekly","monthly","yearly"))) %>%
   arrange(cohort) %>%
   select (extract_date, review, week_57_start, time_period_eligible, tabno, surv, interv, cohort, area_name, area_type, week_no) %>% 
-  filter(interv>=371 & interv<=518) %>% 
-  filter(substr(time_period_eligible,5,10) != "02-MAR") %>% 
-  filter(substr(time_period_eligible,5,10) != "09-MAR") %>% 
-  filter(substr(time_period_eligible,5,10) != "16-MAR") %>% 
-  filter(substr(time_period_eligible,5,10) != "23-MAR")
+  filter(interv>=371 & interv<=518)
```
Comment (vaelliott) [inline app_data_preparation.R:806]: again a wee comment might help as someone new to the data might not realise that interv is a field that represents days and 371 is 53 weeks and i'm assuming this is how we define 13 months?

**Comment by lucindalawrie** [comment]
Ah, thanks Vicky! I knew there would be something I had forgotten to update. I'll also add some comments where you have suggested above. Thanks for reviewing 😄

**Pull request merged**